cosmonic/awesome-cosmonic
# update message to point to next steps

## Primary Reviewer
@brooksmtownsend 

## Feature or Problem
Our hello world actor is great, but it would be even better if by default it gave users a sense of their next steps

For now I included a link to https://cosmonic.com/docs/to-build/image-generator, but I'm open to other ideas

### Manual Verification
I ran `cosmo tutorial hello` with this change, and:
```
🎉 That's it! Access your actor securely through a wormhole now:
    curl https://withered-shape-7517.cosmonic.app

❯ curl https://withered-shape-7517.cosmonic.app
Hello, World, from Cosmonic!

Congratulations on deploying this actor! Wondering what to build next?
Check out our Things to Build guide on an XKCD image generator:

https://cosmonic.com/docs/to-build/image-generator
```
